BlockEntity/BEFruitpress.cs

```diff
@@ -230,17 +233,19 @@ private void onTick100msServer(float dt)
             double totalHours = Api.World.Calendar.TotalHours;
 
             double squeezeRel = mashStack.Attributes.GetDouble("squeezeRel", 1);
-            double litresToTransfer = Math.Min(juiceableLitresLeft, (totalHours - lastLiquidTransferTotalHours) * 50f);
+            double litresJuiceable = Math.Max(0, juiceableLitresLeft - ((juiceableLitresLeft + juiceableLitresTransfered) * screwPercent)); //Determines how much of the mash we can juice for how far down the screw is cranked.

```

Comment:
Determines how much of the mash we can juice for how far down the screw is cranked.